dcos_installer/upgrade.py

```diff
@@ -96,6 +115,19 @@
 pkgpanda fetch --repository-url={{ bootstrap_url }} {{ cluster_packages }} > /dev/null
 pkgpanda activate --no-block {{ cluster_packages }} > /dev/null
 
+end=$(date -ud "5 minute" +%s)
+while [[ $(date -u +%s) -le $end ]]
+do
+  if 3dt check node-poststart && 3dt check cluster; then
+     echo "Node and cluster checks passed"
+     exit 0
+  fi
+  echo "Waiting for node and cluster checks to pass"
+  sleep 5
+done
+
+exit 1
```

Comment:
The output of node and cluster checks gets printed out, check the integration test, its actually noisy